packages/lesmis/src/api/queries/useEntityData.js

```diff
@@ -4,11 +4,30 @@
  *
  */
 import { useQuery } from 'react-query';
+import { useHistory } from 'react-router-dom';
 import { get } from '../api';
+import { useUser } from './useUser';
+
+const useValidatedQuery = query => {
```

Comment:
Clever hook! Feels ever so slightly like it should be moved out to a common util place, but if we're only using it here, then it make sense to have it alongside

Reply:
Yeah I actually moved it in one of the next PRs